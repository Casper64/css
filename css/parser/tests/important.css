.test {
    color: !asdf;
    background: !important;
    border: black 1px !important solid;
    padding: 10px !important;
}