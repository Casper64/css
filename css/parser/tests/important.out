css/parser/tests/important.css:2:12: error: expecting "!important"
    1 | .test {
    2 |     color: !asdf;
      |            ~~~~~
    3 |     background: !important;
    4 |     border: black 1px !important solid;
css/parser/tests/important.css:3:17: error: expecting a value before "!important"
    1 | .test {
    2 |     color: !asdf;
    3 |     background: !important;
      |                 ~~~~~~~~~~
    4 |     border: black 1px !important solid;
    5 |     padding: 10px !important;
css/parser/tests/important.css:4:23: error: expecting ";": "!important" has to be the last identifier in a declaration
    2 |     color: !asdf;
    3 |     background: !important;
    4 |     border: black 1px !important solid;
      |                       ~~~~~~~~~~
    5 |     padding: 10px !important;
    6 | }